types/dogapi/dogapi-tests.ts

```diff
@@ -22,11 +22,13 @@ dogapi.event.create(
 // one metric
 // single point without date
 dogapi.metric.send_all(
-    [{
-        metric: 'metricName',
-        points: 500,
-    }],
-    (err: Error | null, res: string) => {}
+    [
+        {
+            metric: 'metricName',
+            points: 500,
+        },
+    ],
```

Comment:
I am not a big fan of merging style fixes & logic fixes all together. Can we do it in another PR if you really unhappy about it please?

Reply:
Overzealous auto-formatting extensions. I'll separate it out.

Reply:
@olebedev please see latest commit.

I will submit further PR to align the formatting of these files with the main repo's formatting https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json